hana/saptune.sls

```diff
@@ -11,4 +11,8 @@ apply_saptune_solution_{{ host }}_{{ name }}:
   saptune.solution_applied:
     - name: {{ saptune_solution }}
 
+start_saptune_daemon_{{ host }}_{{ name }}:
```

Comment:
even that the command option (for now) is called 'daemon', I would prefer to change the label from 'start_saptune_daemon_' to 'start_saptune_service_' as this reflects much better, what the command really does. It starts the systemd service  'tuned.service' (in saptune version 2) and in the future it starts the systemd service 'saptune.service' (saptune version 3)